loopdb/sqlc/queries/batch.sql

```diff
@@ -62,6 +62,29 @@ INSERT INTO sweeps (
         amt = $5,
         completed = $6;
 
+-- name: GetParentBatch :one
+SELECT
+        sweep_batches.*
+FROM
+        sweep_batches
+JOIN
+        sweeps ON sweep_batches.id = sweeps.batch_id
+WHERE
+        sweeps.swap_hash = $1
+AND
+        sweeps.completed = TRUE
```

Comment:
Completed and confirmed because sweeps may be completed but may be part of multiple sweep batches. So we need to also find the one batch that has the sweep and also is confirmed on-chain.